frontend/src/components/FeedbackTextBox.vue

```diff
@@ -0,0 +1,52 @@
+<template lang="pug">
+
+    v-card(class="rounded-card ")
```

Comment:
classes can be added in a better way. Utilize it since we're using pug. ex: `v-card.rounded-card` instead of `v-card(class="rounded-card ")`